models/accessom3.py

```diff
@@ -0,0 +1,85 @@
+"""Specific Access-OM3 Model setup and post-processing"""
+
+import hashlib
+from pathlib import Path
+from payu.models.cesm_cmeps import Runconfig
+from typing import Dict, Any
+
+from models.model import Model
+
+BASE_SCHEMA_URL = "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/access-om2/experiment/reproducibility/checksums"
```

Comment:
I'm kinda surprised (in a good way) that the schema is so versatile it can be (ab)used in this way.

Reply:
Yeah, I chose the use the same schema mainly because it was easiest/fastest at the time. But I think the schema as it currently exists would work for most/all models: did any of the strings in this list change?

If we do decide to use this schema generally it may be worth renaming because users may want to put strings other than checksums in the list, e.g. values such as global sums (or can I still call that a "checksum"??)

Reply:
> If we do decide to use this schema generally it may be worth renaming because users may want to put strings other than checksums in the list, e.g. values such as global sums (or can I still call that a "checksum"??)

Yeah I had thought if it was more generally useful then it (c|s)hould have a less specific name.